modules/ixBidAdapter.js

```diff
@@ -688,10 +689,10 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
         },
       }
 
-      if (ext.dfp_ad_unit_code) {
-        _bannerImpression.ext = {
-          dfp_ad_unit_code: ext.dfp_ad_unit_code
-        }
+      if (ext.dfp_ad_unit_code || gpid) {
+        _bannerImpression.ext = {};
+        _bannerImpression.ext.dfp_ad_unit_code = ext.dfp_ad_unit_code;
+        _bannerImpression.ext.gpid = gpid;
```

Comment:
if gpid is null but dfp_ad_unit_code is not, would this throw an exception?

Reply:
Both `gpid` and `dfp_ad_unit_code` are `undefined` when they don't exist. If either `gpid` or `dfp_ad_unit_code` are `undefined` they will be removed from the `r` object once we run `JSON.stringify`.